Enhance data visualization attributes section with more images (#51)
fhdsl/DataViz_Considerations: 04_graph_info.qmd

#### Example

An example of a Ridgeline plot in the biomedical, bioinformatics, or cancer informatics literature is Figure 6H from @Xing_etal_2025 showing the distribution of various myeloid cell state scores (M1 macrophage, M2 macrophage, angiogenesis, and phagocytosis) following analysis of single cell RNA sequencing data for certain pre-defined gene sets. Specifically, each of these ridgeline plots is comparing the distributions of the cell state scores for Brain metastases (BrM) against those of primary tumors (PT).

```{r fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = "", fig.link = "https://doi.org./10.1016/j.ccell.2025.03.025"}
ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3b3602af4d0_0_60#slide=id.g3b3602af4d0_0_60")
```

> This example highlights the utility of Ridgeline plots in making comparisons between a small number of groups (2 - BrM vs PT) for several categories (4 - the cell state scores), leading to a large number of overall comparisons even if there are not a lot of groups involved in each comparison. Within this example, some of the comparisons show high similarity between the groups (e.g., M1 score), while others depict larger differences (e.g., phagocytic score). Notice the use of faceting to split out ridgeline plots into different panels or subplots for the cell state score categories and the use of color to signify the BrM and PT groups.

Reviewer: ```suggestion
This example highlights the utility of Ridgeline plots in making comparisons between a small number of groups (2 - BrM vs PT) for several categories (4 - the cell state scores), leading to a large number of overall comparisons even if there are only a few groups involved in each comparison. Within this example, some of the comparisons show high similarity between the groups (e.g., M1 score), while others depict larger differences (e.g., phagocytic score). Notice the use of faceting to split out ridgeline plots into different panels or subplots for the cell state score categories and the use of color to signify the BrM and PT groups.
```

```

If another dimension of data were needed, a **bubble chart** could be utilized like this example in Supplemental Figure S5C from @Ford_etal_2024 which uses color to represent the average expression of certain cell type marker genes across cell types. However, it also represents the number of single cells which expressed that marker gene utilizing the size of points. Notice a large red circle represents a gene that is highly expressed in that cell type for many single cells within the sample. Whereas a large tan circle represents a gene that was lowly expressed in that cell type for many single cells within the sample. Finally, a small circle will represent a gene whose expression level (whatever it is) was not observed in many single cells within that sample.

```{r fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = "", fig.link = "https://www.cell.com/iscience/fulltext/S2589-0042(24)01977-1"}
ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3f2e7f70982_0_1446#slide=id.g3f2e7f70982_0_1446")
```

> In this example, the marker genes are represented within the rows and the different cell types are represented within the columns (inverting the order from the previous example.

Reviewer: ```suggestion
In this example, the marker genes are represented within the rows and the different cell types are represented within the columns (inverting the order from the previous example).
```
